standard/types.md

```diff
@@ -396,7 +396,7 @@ There does not need to exist a `System.ValueTuple<...>` declaration that directl
 
 Element names within a tuple type shall be distinct. A tuple element name of the form `ItemX`, where `X` is any sequence of non-`0`-initiated decimal digits that could represent the position of a tuple element, is only permitted at the position denoted by `X`.
 
-The optional element names are not represented in the `ValueTuple<...>` types, and are not stored in the runtime representation of a tuple value. There is an identity conversion between all tuple types with the same arity and identity-convertible sequences of element types, as well as to and from the corresponding constructed `ValueTuple<...>` type.
+The optional element names are not represented in the `ValueTuple<...>` types, and are not stored in the runtime representation of a tuple value. Identity conversions (§10.2.2) exist between tuples with identity-convertible sequences of element types
```

Comment:
```suggestion
The optional element names are not represented in the `ValueTuple<...>` types, and are not stored in the runtime representation of a tuple value. Identity conversions (§10.2.2) exist between tuples with identity-convertible sequences of element types.
```

(Just finishing the sentence with a period.)

Reply:
This loses the statement of an identity conversion between tuples and `ValueTuple<...>` though. Is that because you consider it to be covered two paragraphs above?

Reply:
It does lose that information. I moved it to the section on Identity conversions, which is linked here. I thought that was the better place. (Similar to with the dynamic type, where there's text about identity conversions and refers to the section in Conversions on Identity Conversions